src/js/components/sku-list.js

```diff
@@ -48,21 +49,56 @@ class SkuList extends LitElement {
     this.locale = 'en-US';
     this.type = 'sku';
   }
+
+  /**
+   *
+   * @param {PurchaseDetails} basicSubPurchase
+   * @param {PurchaseDetails} premiumSubPurchase
+   * @param {string} itemId
+   * @return {string}
+   */
+  skuType(basicSubPurchase, premiumSubPurchase, itemId) {
+    if (itemId === BASIC_SUB && premiumSubPurchase) {
+      return 'downgrade';
+    } else if (itemId === PREMIUM_SUB && basicSubPurchase) {
+      return 'upgrade';
+    } else {
+      return this.type;
+    }
+  }
+
   /**
    * @return {TemplateResult}
    */
   render() {
+    const basicSubPurchase = this.purchases.find((purchase) => purchase.itemId === BASIC_SUB);
```

Comment:
Are these just boolean values? Can we maybe set theses to hasBasicSubPurchase?

Reply:
No, these will be the actual `PurchaseDetails` for the basic and premium subs.